proto/ord_schema.proto

```diff
@@ -599,114 +660,126 @@ message Person {
  */
 message Time {
   enum TimeUnit {
-    TIME_UNIT_HOUR = 0;
-    TIME_UNIT_MINUTE = 1;
-    TIME_UNIT_SECOND = 2;
+    TIME_UNIT_UNKNOWN = 0;
```

Comment:
Why the UNKNOWN values for these units? Is this just to have a better default? If so, that's fine but I wanted to check.

Reply:
Yes, sorry for not documenting the reason/issue well